.github/workflows/build.yml

```diff
@@ -34,9 +33,20 @@ defaults:
 env:
   BUILD_ID: ${{ inputs.buildId }}
   BASHBREW_ARCH: ${{ inputs.bashbrewArch }}
+
+  # the image we'll run to access the signing tool
+  # https://explore.ggcr.dev/?repo=docker/image-signer-verifier
+  IMAGE_SIGNER: 'docker/image-signer-verifier:0.6.0@sha256:81f1ba7d3add2d0e9640d58b059ef0435327f68fc104aca4a685e8c0042e0ed6'
+
+  # Docker Hub repository we'll push the (signed) attestation artifacts to
+  REFERRERS_REPO: oisupport/referrers
```

Comment:
Has this repository been created yet?

Reply:
How is the location of the referrers discovered in the verification flow? I could not find an reference to it in the TUF repo.

Reply:
It isn't used yet. It will be used in the next PRs

Reply:
Removed